source/develop/release-management/features/storage/incremental-backup.md

```diff
@@ -246,15 +260,11 @@ POST /vms/vm-uuid/backups
 ```
 
 Response:
-```
+```xml
 <backup id="backup-uuid">
-    <disks>
-        <disk id="disk-uuid" />
-        ...
-        ...
-    </disks>
+    <link href="/ovirt-engine/api/vms/vm-uuid/backups/backup-uuid/disks" rel="disks"/>
     <phase>initiailizing</phase>
-    <creation_date>
+    <creation_date>xxx</creation_date>
```

Comment:
Should be example of a real value like:

    <creation_date>2020-11-29T17:24:49.219+02:00</creation_date>